scripts/tools/run_valgrind_gnitest

```diff
@@ -0,0 +1,115 @@
+#!/usr/bin/env python
+#
+# Run gnitest using valgrind
+#
+
+import argparse
+import sys, subprocess
+
+def check_args(args):
+    return
+
+def get_pattern_list(args, pattern_list):
+    if args.gnitest_pattern != None:
+        pattern_list += [ args.gnitest_pattern ]
+        return
+
+    p = subprocess.Popen([args.gnitest_exec, '--ascii', '--list'],
```

Comment:
There's a --launcher flag which lets you determine which (if any) launcher to use.